test/test_upgrade_database.cpp

```diff
@@ -757,7 +757,7 @@ TEST_IF(Upgrade_Database_9_10, REALM_MAX_BPNODE_SIZE == 4 || REALM_MAX_BPNODE_SI
     if (REALM_MAX_BPNODE_SIZE == 1000) {
         auto sg = DB::create(*hist, temp_copy);
         if (generate_json) {
-            std::ofstream expect(test_util::get_test_path_prefix() + "expect_test_upgrade_database_9_to_10.json");
+            std::ofstream expect(test_util::get_test_path_prefix() + "/expect_test_upgrade_database_9_to_10.json");
```

Comment:
Why this `/` here? Isn't what we do in `bool initialize_test_path(int argc, const char* argv[])` not sufficient?